public/js/movie_page_script.js

```diff
@@ -0,0 +1,135 @@
+const API_key = 'e0b8ee73-8e36-4d46-9651-177fdcb541a8';
+const API_url_filmInfo = 'https://kinopoiskapiunofficial.tech/api/v2.2/films/'
+
+
+// Получение текущего id фильма для поиска информации о нем
+const currFilmId = new URLSearchParams(window.location.search).get('id');
+getMovieInfo(`${API_url_filmInfo}${currFilmId}`);
+
+/**
+ * 
+ * @param {string} url 
+ */
+async function getMovieInfo(url) {
+    const resp = await fetch(url, {
+        headers: {
+            "Content-Type": "application/json",
+            "X-API-KEY": API_key
+        }
+    });
+    const respData = await resp.json();
+    showMovieInfo(respData)
+}
+
+/**
+ * Если фильм новый, то изменяет заголовок на 'Рейтинг ожидания'
+ * 
+ * @param {string} rate 
+ * @param {string} rateAwait
+ * @returns {string}
+ */
+function checkExpRating(rate, rateAwait) {
+    if (rate == null && rateAwait != null) {
```

Comment:
Лучше сравнивать без приведения типов
```suggestion
    if (rate === null && rateAwait !== null) {
```

Либо можно сделать проще и написать так
```suggestion
    if (rate == null && rateAwait != null) {
```
```suggestion
    if (!rate && rateAwait) {
```